pages/common/gradle.md

```diff
@@ -0,0 +1,19 @@
+# Gradle
+
+> Gradle is the official build system for Android Studio
+
+- Show version
+
+`gradle -v`
+
+- Complie a package(Make sure you have a gradle file in the workspace)
```

Comment:
I think the comment in parens is unnecessary. Also you have a typo in `Complie`.